javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,22 @@
+var $spans = document.querySelectorAll('span');
+var i = 0;
+
+function typingEvent(event) {
+  if (!$spans[i]) {
+    if (i === 0) {
+      $spans[i].className = 'active';
+    }
+    return;
+  }
+
+  var $currentLetter = document.querySelector('.active');
```

Comment:
Rather than query the DOM again, you should be able to mark the current letter and then increment your `i` variable.